hie-compat/hie-compat.cabal

```diff
@@ -24,7 +24,7 @@ library
   default-language:    Haskell2010
   build-depends:
      base < 4.17, array, bytestring, containers, directory, filepath, transformers
-  if flag(ghc-lib)
+  if flag(ghc-lib) && impl(ghc < 9)
```

Comment:
I'm surprised by this change. It was working fine before.

Reply:
It was a rebase from @pepeiborra branch

Reply:
@guibou this is resolved, @pepeiborra have been merged into the master branch